rules/0.3-allow-points-transfer.py

```diff
@@ -33,7 +36,7 @@ def should_allow(pr):
 
         total_points_change += points_change
 
-    if total_points_change >= 0:
-        raise Exception('points change PRs must on net remove points')
+    if total_points_change > 0:
+        raise Exception('points change PRs cannot net create points')
```

Comment:
The original check here was not a bug. Players receive 1 point for having their PR merged, which means that a PR which has 0 net points from transfers has a total net of 1 point.

Reply:
While I don't disagree conceptually, the `total_points_change` variable here is only the sum of the bonus amounts in files, per user. Whether a merged PR is 1 point or 10 points or zero points, it won't contribute to this logic.

Reply:
It contributes to the logic by informing the threshold. We decided that points changed PRs must be net negative because of it. Why should the threshold instead be 0?

Reply:
I see what you're saying, the file reads -1 because the player then starts at 0 after the merge. If that's the intention lets update the README to tell players to start with a -1 bonus PR.

Reply:
It's okay that new players get 1 point for merging their join PR.